openshift/e2e-tests-openshift.sh

```diff
@@ -127,17 +127,18 @@ function timeout() {
 }
 
 function install_servicemesh(){
-  header "Installing ServiceMesh"
+  # header "Installing ServiceMesh"
 
   # Install the ServiceMesh Operator
-  oc apply -f https://raw.githubusercontent.com/openshift/knative-serving/$SERVING_RELEASE/openshift/servicemesh/operator-install.yaml
+  # oc apply -f https://raw.githubusercontent.com/openshift/knative-serving/$SERVING_RELEASE/openshift/servicemesh/operator-install.yaml
```

Comment:
Can we remove it instead of commenting out ? Where is this operator now installed ?